Sources/DeepLearning/Layer.swift

```diff
@@ -1009,12 +1009,32 @@ public struct AvgPool2D<Scalar: TensorFlowFloatingPoint>: Layer {
     }
 }
 
+
+/// A global average pooling layer for temporal data.
+@_fixed_layout
+public struct GlobalAveragePooling1D<Scalar: TensorFlowFloatingPoint>: Layer {
+    /// Creates a global average pooling layer in 1 dimension
```

Comment:
```suggestion
    /// Creates a global average pooling layer in 1 dimension.
```

Please end sentences with a period.